code/game/machinery/alarm.dm

```diff
@@ -635,17 +635,12 @@
 		return
 
 	if ( (get_dist(src, user) > 1 ))
```

Comment:
I'm just not sure how the following fixes the bug? Also to be honest in a fix of this bug I'd expect this get_dist to be changed to in_range, not to restrict you to that if the current fix makes sense.